packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasEstimates.ts

```diff
@@ -70,11 +48,13 @@ export function useGasEstimates({
   gasEstimates: GasEstimates | DepositGasEstimates | undefined
   error: any
 } {
+  const { data: signer } = useSigner()
+
```

Comment:
Have `sourceChainProvider` and `destinationChainProvider` here instead of the fetcher, saving some calls